### PR TITLE
Put a space between after_jump commands

### PR DESCRIPTION
**Pull request by rallytime**
Refs #24226

### CODE REVIEW

File: salt/modules/iptables.py [hunk 1]
```diff
@@ -390,7 +390,7 @@ def build_rule(table=None, chain=None, command=None, position='', full=None, fam
             rule += '--{0} {1} '.format(item, kwargs[item])
 
     for item in after_jump:
```
Comment (thusoy) [inline salt/modules/iptables.py]: You need to delete this loop, otherwise you'll be adding all the items after the jump `len(after_jump)` times.

**Comment by rallytime** [comment]
@thusoy How's that now?

**Comment by thusoy** [comment]
@rallytime Deindent four spaces and we're good!

**Comment by rallytime** [comment]
@thusoy Figured I'd get it eventually. Thanks for the help.

**Pull request merged**